src/app/shared/styles/tl-variables.scss

```diff
@@ -50,7 +50,7 @@
 
   --wvr-font-family-sans-serif: var(--tl-font-family-sans-serif);
 
-  font-family: var(--tl-font-family-sans-serif);
+  font-family: var(--tl-default-font-family-sans-serif);
```

Comment:
Can this be reverted to `--tl-font-family-sans-serif`

Reply:
this had already been updated - i'll redo the pull request - there is an outdated label on there